tests/puppeteer/pages/BO/commonPage.js

```diff
@@ -1,6 +1,6 @@
 const CommonPage = require('../commonPage');
 
-module.exports = class BO_COMMONPAGE extends CommonPage {
+module.exports = class COMMONPAGE extends CommonPage {
```

Comment:
```suggestion
module.exports = class BasePage extends CommonPage {
```
We need to change the class names, not in full uppercase, juste `BasePage`. Following PSR-1 standards, we must write class names in `StudlyCaps` (https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-1-basic-coding-standard.md#3-namespace-and-class-names).
Since `CommonPage` is already taken, we need to change the name. I'm thinking about `BOBasePage` for example ?